neurom/view/view.py

```diff
@@ -722,8 +722,8 @@ def dendrogram(obj, show_diameters=True, new_fig=True, new_axes=True, subplot=Fa
     kwargs['xlim'] = [- dnd.dims[0][0] * 0.5, dnd.dims[-1][0] * 0.5 + displacement]
 
     kwargs['title'] = kwargs.get('title', 'Morphology Dendrogram')
-    kwargs['xlabel'] = kwargs.get('xlabel', '')
-    kwargs['ylabel'] = kwargs.get('ylabel', '')
+    kwargs['xlabel'] = kwargs.get('xlabel', 'micrometers (um)')
+    kwargs['ylabel'] = kwargs.get('ylabel', 'micrometers (um)')
```

Comment:
I think we shouldn't make assumptions if the format doesn't specify micrometers. If we don't know for sure what the units are, then we don't say anything and leave it up to the user to set the units. But this process should be made as easy as possible, and it should be clear to the user that they are expected to do this. For example, setting up a default of "unknown units" in the interface might help.